fdbclient/include/fdbclient/TenantManagement.actor.h

```diff
@@ -457,8 +457,10 @@ Future<Void> configureTenantTransaction(Transaction tr,
 		}
 		if (originalEntry.tenantGroup.present()) {
 			// Remove this tenant from the original tenant group index
-			TenantMetadata::tenantGroupTenantIndex().erase(
-			    tr, Tuple::makeTuple(originalEntry.tenantGroup.get(), updatedTenantEntry.id));
+			TenantMetadata::tenantGroupTenantIndex().erase(tr,
+			                                               Tuple::makeTuple(originalEntry.tenantGroup.get(),
+			                                                                updatedTenantEntry.tenantName,
```

Comment:
Can configure change the tenant name? Would it be better to use `originalEntry.tenantName` for deletion instead?

Reply:
It can't change the name, but we could still change this code